core/src/completed_data_sets_service.rs

```diff
@@ -1,3 +1,9 @@
+//! [`CompletedDataSetsService`] is a hub, that runs different operations on when a "completed data
+//! set", also known as a [`Vec<Entry>`] is received by the validator.
```

Comment:
`"completed data set"` -> `CompletedDataSet`

Reply:
There is no type called `CompletedDataSet`.
There is a `CompletedDataSetInfo`, but that is different.
Here I am referring to the collection of shards that form a `Vec<Entry>`.